### ISSUE TITLE
no cursor in the searchbar

### ISSUE THREAD

**Issue by fossfreedom** [libredeb/lightpad#5]
testing with Arc gtk theme and others i note I dont see a cursor in the searchbar which kind of confused me first since this would normally mean "type here".

Ubuntu Budgie 18.04

**Comment by libredeb** [comment]
@fossfreedom yes, because it is a port that I am carrying that was originally written in GTK+ 2.0 it is something that is purposely programmed like this... I understand that there should be a cursor there, we are working on it.

**Comment by libredeb** [comment]
@fossfreedom done! 😊 The issue was resolved in this commit [751dc7a](https://github.com/libredeb/lightpad/commit/751dc7adf0a24457d82cd37cb41f0325e261b362)
